mqt/bench/benchmark_generator.py

```diff
@@ -347,6 +342,9 @@ def generate_target_dep_level_circuit(
     return num_generated_benchmarks != 0
 
 
+ERROR_MSG = "\n Problem occurred in outer loop: "
```

Comment:
Is it standard procedure, to have some variable declaration between methods? To me this feels a little wrong because I thought all functionality should be either included in the method where it is used or in a `utils` file with the respective getter method?

Reply:
To be honest, this was suggested to me by PyCharm. The real underlying problem here is that the same string was repeated roughly 8 times. That already tells me that the underlying design could benefit from some refactoring that extracts common functionality. However, I didn't dare to start the refactoring process as part of this PR.
As such, I would consider the current position of this variable intermediate at best.
If you are still worried about it, I'd move it.

Just as a side note: I really despise `utils` files. They frequently turn out to be some Frankenstein monster of mostly unrelated code that would be much better suited as a member function of a class or something similar where it is much more coupled to its intent.